src/themes/components/carousel.ts

```diff
@@ -7,20 +7,58 @@ const [r, g, b] = hexToRgb(colors.gray[900]);
 
 const parts = [
   'container',
+  'carousel',
   'slide',
   'slideContainer',
   'buttonContainer',
   'button',
   'icon',
+  'pagination',
+  'paginationButtonContainer',
+  'paginationButton',
 ];
 
 const fullScreen = {
   container: {
-    height: '100%',
+    backgroundColor: 'black',
+    position: {
+      base: 'fixed',
+      md: 'static',
+    },
+  },
+  carousel: {
     paddingX: {
       md: '5xl',
     },
-    backgroundColor: 'black',
+  },
+  pagination: {
+    overflow: 'hidden',
+    position: 'relative',
+    paddingX: {
+      base: 'md',
+      md: '5xl',
+    },
+    height: '7.5rem',
+  },
+  paginationButtonContainer: {
+    position: 'absolute',
+    top: '0',
+    width: 'lg',
+    height: 'full',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    color: 'white',
+    backgroundColor: 'transparent',
+  },
+  paginationButton: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    width: 'md',
+    height: 'md',
+    borderRadius: 'sm',
+    backgroundColor: `rgba(${r},${g},${b},0.8)`,
```

Comment:
Would it be possible to use the opacity form the tokens instead of converting the colour?

Reply:
No. If we set opacity on the element all children are opaque as well and it's not what we want. An alternative to converting the colour is using absolute positioning and adding complexity to the DOM and styling that we don't need. This is simpler.

Reply:
no because this would also blur the icon itself and not only the background